ang/crmUi.js

```diff
@@ -1214,6 +1214,13 @@
                     ++h1Count;
                   }
                 });
+                // Standalone
+                const standaloneH1 = document.querySelector('h1.crm-page-title');
+                if (standaloneH1) {
+                  standaloneH1.innerHTML = newPageTitle;
+                  $el.hide();
+                  ++h1Count;
+                }
```

Comment:
Maybe too clever, but another way to do this would be simply change L1210 to:
```
$('h1:not(.crm-container h1), .crm-page-title-wrapper>h1').each(function() {
```